cypress/e2e/awx/inventories-constructed/inventoriesConstructed.cy.ts

```diff
@@ -62,7 +62,19 @@ describe('Constructed Inventories CRUD Tests', () => {
     cy.singleSelectBy('[data-cy="organization"]', organization.name);
     // this can be simplified if we include data-cy to the search button of instance groups
     cy.multiSelectByDataCy('instance-group-select-form-group', [instanceGroup.name]);
-    cy.multiSelectByDataCy('inventories', [inventory.name]);
+
```

Comment:
I believe we don't need the cy.intercepts or the multiple inventory names

Reply:
So should I remove lines 64 and 65?

Reply:
I believe we can remove lines 65-77, and add back ```cy.multiSelectByDataCy('inventories', [inventory.name]);``` after line 64!